presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveFileFormats.java

```diff
@@ -131,10 +135,47 @@ public void testSequenceFile(int rowCount)
 
         assertThatFileFormat(SEQUENCEFILE)
                 .withColumns(testColumns)
+            .withRowsCount(rowCount)
+            .isReadableByRecordCursor(new GenericHiveRecordCursorProvider(HDFS_ENVIRONMENT));
+    }
+
+    @Test(dataProvider = "rowCount")
+    public void testCsvFile(int rowCount)
+            throws Exception
+    {
+        Set<String> csvSupportingColumns = ImmutableSet.of("t_empty_string", "t_string");
```

Comment:
I think we can also handle
* `t_empty_varchar`
* `t_varchar`
* `t_varchar_max_length`
* `t_null_string`
* `t_null_varchar`

Reply:
We might want to build the list like this:
```java
List<TestColumn> testColumns = TEST_COLUMNS.stream()
        .filter(column -> column.isPartitionKey() || isUnboundedVarchar(column.getType()))
        ...

private static boolean isUnboundedVarchar(String hiveTypeName)
{
    return HiveType.valueOf(hiveTypeName).getType(TYPE_MANAGER).equals(VARCHAR);
}
```